index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Atom</title>
+    <link href="styles2.css" rel="stylesheet" />
+    <script
+      src="https://kit.fontawesome.com/d39c91b79b.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+
+  <body>
+    <!-- Navigation -->
+
+    <nav class="navigation">
+      <ul class="nav-links">
+        <li class="nav-home">Home</li>
+        <li class="nav-documents"><a href="/documents.html">Documents</a></li>
+        <!-- <li class="nav-component"><a href="/components.html">Components</a></li> -->
+        <!--*components*-->
+      </ul>
+    </nav>
+
+    <div class="image-and-description">
+      <!-- Hero-Image -->
+      <header class="header-image-container">
+        <img class="header-image" src="images/atom1.png" />
+      </header>
+
+      <!-- Heading-Hero -->
+      <h1 class="main-heading">Atom - <span class="ui">UI</span></h1>
+
+      <!-- Description -->
+
+      <div class="atom-description">
+        <p>
+          Build easily fast and responsive mobile-first web-applications with
+          Atom. It is light, easy to set up and comes with extensive prebuilt
+          components.
+        </p>
+      </div>
+    </div>
+
+    <div class="installation">
+      <h2 class="installation-heading">Installation</h2>
+      <div>
+        Install Atom's source via the link provided below. Directly link the
+        below code to the head of your html file.
+      </div>
+    </div>
+
+    <div class="installation-guide">
+      <h2 class="installation-heading">CSS Installation Link</h2>
+      <iframe
+        class="link-iframe"
+        src="https://carbon.now.sh/embed?bg=rgba%28248%2C231%2C28%2C1%29&t=blackboard&wt=none&l=auto&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Fira+Code&fs=14px&lh=152%25&si=false&es=2x&wm=false&code=%250A%2520%253Clink%2520href%253D%2522styles2.css%2522%2520rel%253D%2522stylesheet%2522%2520%252F%253E"
+        style="
+          width: 100%;
+          height: auto;
+          border: 0;
+          transform: scale(1);
+          overflow: scroll;
+        "
+        sandbox="allow-scripts allow-same-origin"
+      >
+      </iframe>
+    </div>
+
+    <!-- Footer -->
+    <div class="footer-box">
+      <footer class="footer-ele">
```

Comment:
give a well understandable and readable class name even if it becomes bigger. write completely like footer-element or anything you want.